schedctl/config.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"github.com/bgzzz/go-schedule/common"
+
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+)
+
+var Config *SchedCtlConfig
```

Comment:
Same question as in the "management node" part - any good reason this is a global?

Reply:
done